generator/src/main/resources/line-bot-sdk-ruby-generator/api.pebble

```diff
@@ -55,12 +55,10 @@ module Line
           {%- for op in operations.operation %}
 
           # {{ op.notes }}
+          # This requests to <tt>{{ op.httpMethod }} {{ endpoint(operations.classname) }}{{ op.path }}</tt>
           #
-          {%  if op.summary -%}
-          # @summary {{ op.summary }}
-          {%  endif -%}
```

Comment:
@summary doesn't exist, and `op.summary` is rarely used. This change deletes this.